crates/sdk/src/keygen/mod.rs

```diff
@@ -352,6 +452,7 @@ pub fn leaf_keygen(
     let leaf_engine = BabyBearPoseidon2Engine::new(fri_params);
     let leaf_vm_pk = info_span!("keygen", group = "leaf")
         .in_scope(|| VirtualMachine::new(leaf_engine, leaf_vm_config.clone()).keygen());
+    check_recursive_verifier_size(&leaf_vm_pk.get_vk(), fri_params, fri_params.log_blowup);
```

Comment:
the log_blowup here is not next, but also this function is only used in a weird benchmarking codepath, so we could just remove this line